pkg/vmprovider/providers/vsphere2/vmlifecycle/update_status.go

```diff
@@ -267,9 +266,31 @@ func convertNetIPRouteConfigInfo(routeConfig *types.NetIpRouteConfigInfo) []vmop
 		return nil
 	}
 
-	// TODO: Prob only want to show default routes. Will be very verbose on TKG nodes.
-	out := make([]vmopv1.VirtualMachineNetworkIPRouteStatus, 0, len(routeConfig.IpRoute))
+	// Try to skip uninteresting routes so the Status isn't as cluttered.
```

Comment:
I am fine skipping loopback addresses, but I do think static routes should be shown. There are often non-default routes.

Reply:
Let me be more clear, I'm also fine skipping link-local routes.